INSTALL.md

```diff
@@ -1,43 +1,139 @@
+## Autoenv module
 
+[Install Autoenv](https://github.com/kennethreitz/autoenv#install) however
+you'd like. (We use [Homebrew]((http://brew.sh)):
+
+Run:
+```bash
+brew install autoenv
+```
+
+After installation, Homebrew will output instructions similar to:
+```bash
+To finish the installation, source activate.sh in your shell:
+  source /Users/[YOUR MAC OSX USERNAME]/homebrew/opt/autoenv/activate.sh
+```
+
+Any time you run the project you'll need to run that last line. If you'll
+be working with the project consistently, we suggest adding it to your bash
+profile by running:
+```bash
+echo 'source /Users/[YOUR MAC OSX USERNAME]/homebrew/opt/autoenv/activate.sh' >> ~/.bash_profile
+```
+
+If you need to find this info again later, you can run:
+```bash
+brew info autoenv
+```
+
+## Elasticsearch
+
+[Install Elasticsearch](http://www.elastic.co/guide/en/elasticsearch/reference/current/setup.html)
+however you'd like. (We use [Homebrew](http://brew.sh)):
+```bash
+brew install elasticsearch
+```
+
+Just as with autoenv, Homebrew will output similar instructions after
+installation:
+```bash
+To have launchd start elasticsearch at login:
+    ln -sfv /Users/[YOUR MAC OSX USERNAME]/homebrew/opt/elasticsearch/*.plist ~/Library/LaunchAgents
+Then to load elasticsearch now:
+    launchctl load ~/Library/LaunchAgents/homebrew.mxcl.elasticsearch.plist
+Or, if you don't want/need launchctl, you can just run:
+    elasticsearch --config=/Users/[YOUR MAC OSX USERNAME]/homebrew/opt/elasticsearch/config/elasticsearch.yml
+```
+
+Any time you work on the projct, you'll need to open a new tab and
+run that last line. If you'll be working on the project consistently,
+we suggest using the first option utilizing `launchd`.
+
+If you need to find this info again later, you can run:
 ```bash
-$ workon cfgov-refresh
+brew info elasticsearch
+```
+
+## Sheer
 
-$ pip install git+git://github.com/dpford/flask-govdelivery
-$ pip install git+git://github.com/rosskarchner/govdelivery
+To [install sheer](https://github.com/cfpb/sheer#installation), start by
+cloning the sheer GitHub project to wherever you keep your projects
+(not inside cfgov-refresh):
+```bash
+git clone https://github.com/cfpb/sheer.git
 ```
 
-## Back-end environment variables
+Create a virtualenv for sheer, which you'll name `cfgov-refresh`:
+```bash
+mkvirtualenv cfgov-refresh
+```
 
-The project needs a number of environment variables.
-The project uses a WordPress API URL to pull in content
-and GovDelivery for running the subscription forms:
+The new virtualenv will activate right away. To activate it later on
+(say, in a new terminal session) use the command `workon cfgov-refresh`.
+You'll know you have a virtual environment activated if you see the
+name of it in parentheses before your terminal prompt. Ex:
+```bash
+(cfgov-refresh)username$
+```
+
+Install sheer into the virtualenv with the `-e` flag (which allows you
+to make changes to sheer itself). The path to sheer is the root
```

Comment:
Should've been explicit that Sheer needed to be capitalized on many lines throughout the file :)